peacebot/core/plugins/Moderation/mod.py

```diff
@@ -0,0 +1,259 @@
+    await member.ban(
+        delete_message_days=ctx.options.days or 0, reason=ctx.options.reason
+    )
+    embed = (
+        hikari.Embed(
+            description=f"🔨 Banned -> {member}\n**Reason:** {ctx.options.reason}",
+            color=EmbedColors.ERROR,
+            timestamp=datetime.now().astimezone(),
+        )
+        .set_author(
+            name=f"{ctx.author}(ID {ctx.author.id})", icon=ctx.author.avatar_url
+        )
+        .set_thumbnail(member.avatar_url)
+    )
+    await mod_logs.send(embed=embed)
+    response = await ctx.respond(embed=embed)
+    message_link = (await response.message()).make_link(ctx.guild_id)
+    await register_cases(
+        guild_id=ctx.guild_id,
+        moderator=ctx.author.id,
+        target=member.id,
+        reason=ctx.options.reason,
+        message_link=message_link,
+        channel_id=ctx.channel_id,
+        type="Ban",
+    )
+
+
+@mod_plugin.command
+@lightbulb.command("case", "Group for case commands")
+@lightbulb.implements(lightbulb.SlashCommandGroup, lightbulb.PrefixCommandGroup)
+async def case_command(ctx: lightbulb.Context) -> None:
+    await ctx.bot.help_command.send_group_help(ctx, ctx.command)
+
+
+@case_command.child
+@lightbulb.option("case_id", "Case ID to look for")
+@lightbulb.command("search", "Get a specific case")
+@lightbulb.implements(lightbulb.SlashSubCommand, lightbulb.PrefixSubCommand)
+async def case_search_command(ctx: lightbulb.Context) -> None:
+    model = await ModLogs.get_or_none(guild_id=ctx.guild_id, id=ctx.options.case_id)
+    if model is None:
+        raise ModerationError("No case found with that ID.")
+    embed = hikari.Embed(
+        title=f"{model.type} Case | No. {model.id}",
+        description=f"[Jump to Message!]({model.message})",
+        color=EmbedColors.INFO,
+    )
+    fields: list[tuple[str, str | int, bool]] = [
+        ("Moderator", model.moderator, True),
+        ("Target", model.target, True),
+        ("Reason", model.reason, True),
+        ("Channel", model.channel, True),
+        ("Guild ID", model.guild_id, True),
+        (
+            "Time of Action",
+            f"<t:{int(model.timestamp.timestamp())}:F> • <t:{int(model.timestamp.timestamp())}:R>",
+            False,
+        ),
+    ]
+    for name, value, inline in fields:
+        embed.add_field(name=name, value=value, inline=inline)
+
+    await ctx.respond(embed=embed)
+
+
+@case_command.child
+@lightbulb.command("list", "List all the Moderation Cases")
+@lightbulb.implements(lightbulb.SlashSubCommand, lightbulb.PrefixSubCommand)
+@moderation_role_check
+async def case_list_command(ctx: lightbulb.Context) -> None:
+    case_model = await ModLogs.filter(guild_id=ctx.guild_id)
+    if not len(case_model):
+        raise ModerationError("No Cases could be found for the guild!")
+
+    cases = [
+        f"#{model.id} - <t:{int(model.timestamp.timestamp())}:R> -> {model.moderator}\n**Type**: ```{model.type}```"
```

Comment:
I would suggest you to rather declare a string, and use `.format` method for passing in the values rather than making this long a f-string.

Reply:
Will do, I also was thinking about using `format` instead of f-strings.